DATA_PREPARATION.md

```diff
@@ -158,4 +157,56 @@ Below is an example of multitask dataset:
 ]
 ```
 
+For `key_value_pair` dataset, an additional field `schema` is required to define the task and the label format. Below is an example of its dataset information dictionary:
+
+```json
+[
+    {
+        "name": "multi-image-question-answer",
+        "version": 1,
+        "description": "Answer question related to one or more images",
+        "type": "key_value_pair",
+        "format": "coco",
+        "root_folder": "key_value_pair/multi_img_qa_20240723",
+        "schema": {
+            "name": "Multi-image QA schema",
+            "description": "Provide answer and a rationale.",
+            "fieldSchema": {
+                "answer": {
+                    "type": "string",
+                    "description": "answer."  
+                },
+                "rationale": {
+                    "type": "string",
+                    "description" :"rationale of answer"
+                }
+            }    
+        },
+        "train": {
+            "index_path": "train.json",
+            "files_for_local_usage": [
+                "train_images.zip"
+            ]
+        },
+        "val": {
+            "index_path": "val.json",
+            "files_for_local_usage": [
+                "val_images.zip"
+            ]
+        }
+    }
+]
+```
+The `fieldSchema` is a dictionary mapping each field to a dictionary called `FieldSchema`. In above example, there are two fields: `answer` and `rationale`. Below is the definition of `FieldSchema`:
+
+| Property | Type | Example | Required? |
```

Comment:
nit: There are markdown lint tools from vscode that can help you make the table prettier

Reply:
nice! formatted.